### PR TITLE
Added a short help file for class Pdiff

### PR DESCRIPTION
**Pull request by mhampton**
## Types of changes

- Documentation
- Bug fix
- New feature
- Breaking change

## To-do list

- [ ] Code is tested
- [ ] All tests are passing
- [ ] Updated documentation
- [ ] This PR is ready for review

### CODE REVIEW

File: HelpSource/Classes/Pdiff.schelp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class:: Pdiff
+summary:: returns the difference between the current and previous values of an enclosed pattern
+related:: Classes/Padd
```
Comment (jamshark70) [inline HelpSource/Classes/Pdiff.schelp]: Padd is not really related (different purpose).

The inverse operation of differentiation is integration. It isn't obvious at first, but in pattern-land, the integrator is Pseries. So I'd put this one in as the related class.

Comment (mhampton) [reply HelpSource/Classes/Pdiff.schelp]: Thanks!  That makes a lot of sense, I have changed it.  I also added a file for Pdrop.

**Comment by jamshark70** [comment]
> That makes a lot of sense, I have changed it. I also added a file for Pdrop.

I don't see the change (Padd --> Pseries) registered in the new commit. Could you check again?

Ideally, method documentation (`*new`) should list the arguments too. Pdiff and Pdrop are fairly simple so I won't be too picky, but Pdrop would benefit from a couple of `argument::` tags.

For procedure, usually we want to avoid creeping scope in PRs -- add some help files, let them be reviewed and merged. Other help files can go into a different PR.

**Comment by brianlheim** [comment]
closing as stale, please address requested changes and reopen if you'd like to continue working on this @mhampton !

**Comment by brianlheim** [comment]
whoops, sorry about that, i see changes were (mostly) addressed. i think this is good as-is. @jamshark70 are you ok merging it now or would you prefer to wait until @mhampton adds argument tags as requested?

**Comment by brianlheim** [comment]
ok, thanks @mhampton , @jamshark70 !

**Pull request merged**